### PR TITLE
Update cli_quickstart.hpp

### PR DESCRIPTION
**Pull request by bkmgit**
Add link to Julia page

### CODE REVIEW

File: doc/guide/cli_quickstart.hpp [hunk 1]
```diff
@@ -10,7 +10,7 @@ This page describes how you can quickly get started using mlpack from the
 command-line and gives a few examples of usage, and pointers to deeper
 documentation.
 
-This quickstart guide is also available for @ref python_quickstart "Python".
+This quickstart guide is also available for @ref python_quickstart "Python" and @ref julia_quickstart "Julia".
```
Comment (birm) [inline doc/guide/cli_quickstart.hpp:13]: Pedantic, but let's try to keep it 80 lines.
```suggestion
This quickstart guide is also available for @ref python_quickstart "Python" and
@ref julia_quickstart "Julia".
```

File: doc/guide/python_quickstart.hpp [hunk 2]
```diff
@@ -9,7 +9,8 @@
 This page describes how you can quickly get started using mlpack from Python and
 gives a few examples of usage, and pointers to deeper documentation.
 
-This quickstart guide is also available for @ref cli_quickstart "the command-line".
+This quickstart guide is also available for 
+@ref cli_quickstart "the command-line" and @ref julia_quickstart "Julia"..
```
Comment (birm) [inline doc/guide/python_quickstart.hpp:13]:
```suggestion
@ref cli_quickstart "the command-line" and @ref julia_quickstart "Julia".
```

File: doc/guide/cli_quickstart.hpp (hunk 1 above)
Comment (bkmgit) [reply doc/guide/cli_quickstart.hpp:13]: Thanks.

**Comment by rcurtin** [comment]
Thank you @bkmgit! :+1:

**Pull request merged**